ads/google/a4a/utils.js

```diff
@@ -92,6 +92,9 @@ export let NameframeExperimentConfig;
  */
 export const TRUNCATION_PARAM = {name: 'trunc', value: '1'};
 
+/** @const {Object} */
+const CDN_REGEXP = /^https:\/\/([a-zA-Z0-9_-]+\.)?cdn\.ampproject\.org/;
```

Comment:
This looks correct: host names can be `[.a-zA-Z0-9-]` but in practice sometimes `_` is allowed and `.` converts to `-` in making cache urls.